tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1721,19 +1813,26 @@ def adjust_gamma(image, gamma=1, gain=1):
   pixelwise according to the equation `Out = gain * In**gamma`,
   and then converts the back to the original data type.
 
+  Usage Example:
+    >>> x = [[[1.0, 2.0, 3.0],
+    ...       [4.0, 5.0, 6.0]],      
+    ...     [[7.0, 8.0, 9.0],      
+    ...       [10.0, 11.0, 12.0]]]
+    >>> tf.image.adjust_gamma(x, 0.2)
+    array([[[1.       , 1.1486983, 1.245731 ],
+        [1.319508 , 1.3797296, 1.4309691]],
```

Comment:
Here too, no need for more precision than the first two significant figures

Reply:
Fixed -> https://github.com/tensorflow/tensorflow/pull/35388/commits/76b96090d0c1d93a82c632ac8546b861e9af36ed